addons/vehicles/initSettings.sqf

```diff
@@ -0,0 +1,7 @@
+
+[QGVAR(keepEngineRunning), "CHECKBOX",
+[LSTRING(SettingKeepEngineRunningName), LSTRING(SettingKeepEngineRunningDesc)],
+localize ELSTRING(common,ACEKeybindCategoryVehicles),
+true, // default value
```

Comment:
is it really necessary to make this by default? You can always exit with `eject` action to keep engine off. With this setting `get out` would became the same as `eject`.

Reply:
Eject does very different things for aircraft.

Reply:
I tend to agree with @Dystopian unless we also remove the Eject action in ACE3. Then we could have 1 setting that toggles between:
- Vanilla
- Removed Eject action & Disabled automatic shut-off

Reply:
> Eject does very different things for aircraft.

make it for aircraft only, why force for any vehicles?

Reply:
I'd be happy to have Eject as an aircraft only action.

Reply:
I can agree on the by default disabled setting. But I don't see why it should be aircraft only. This constant turning-off the engine stuff is a disease in the arma mill-sim world. I can agree on wheeled vehicles with a regular piston engine that you constantly turn them on and off but turning off the gas turbine of a tank is just not that common. Maybe we can agree that we make this a setting as well, either all vehicles or just aircrafts?
Also I see a difference between eject and get out even with the keep the engine running feature. Eject allows you to get out of a moving vehicle, get out doesn't. For someone like me who uses keybinds for both it is crucial difference. The get out keybind is close to the often used keys, the eject is on the far end of the keyboard.

Reply:
@brainslush Eject means ejection seat in aircraft, not just keeping the engine running. I am not sure if you missed that part.